src/django/api/fixtures/contributors.json

```diff
@@ -1,1377 +1,1374 @@
 [
     {
         "model": "api.contributor",
-        "pk": 1,
```

Comment:
In 1073cd3 we intentionally make it so that the `pk` of the contributor and `admin_id` were not the same value, which makes it easier to catch bogs related to using the wrong ID.

We will want to make sure our `pk`s continue to start from 1

Because the updated URL does not show up in any other fixtures, rather than committing new fixtures we could instead just find-and-replace `https://info.openapparel.org"` with `https://www.example.com`